runner/run.go

```diff
@@ -42,6 +45,12 @@ func Run(ctx context.Context, wg *sync.WaitGroup, globalCollectionOpts state.Col
 		return
 	}
 
+	driverCleanup, err = pgxv5.RegisterDriver("cloudsql-postgres", cloudsqlconn.WithIAMAuthN())
```

Comment:
I need to return this driverCleanup to the main.go, since the cleanup should happen over there. Also, registering driver needs to happen only one time per the process, therefore it lives here.

https://github.com/GoogleCloudPlatform/cloud-sql-go-connector?tab=readme-ov-file#using-the-dialer-with-databasesql